hazelcast/src/main/java/com/hazelcast/jmx/suppliers/StatSupplier.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jmx.suppliers;
+
+/**
+ * Interface used in MBeans for retrieving the Local Stats
+ * @param <T> LocalStats type
+ */
+public interface StatSupplier<T> {
```

Comment:
Isn't `StatsSupplier` more suitable name? if you think what it returns is `LocalStats`...